src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/services/GetRiskPredictorScoresForPersonServiceTest.kt

```diff
@@ -0,0 +1,144 @@
+package uk.gov.justice.digital.hmpps.hmppsintegrationapi.services
+
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.matchers.collections.shouldHaveSize
+import io.kotest.matchers.shouldBe
+import org.mockito.Mockito
+import org.mockito.internal.verification.VerificationModeFactory
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.whenever
+import org.springframework.boot.test.context.ConfigDataApplicationContextInitializer
+import org.springframework.boot.test.mock.mockito.MockBean
+import org.springframework.test.context.ContextConfiguration
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.gateways.AssessRisksAndNeedsGateway
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.GeneralPredictorScore
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Identifiers
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Person
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Response
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.RiskPredictorScore
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.UpstreamApi
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.UpstreamApiError
+
+@ContextConfiguration(
+  initializers = [ConfigDataApplicationContextInitializer::class],
+  classes = [GetRiskPredictorScoresForPersonService::class],
+)
+internal class GetRiskPredictorScoresForPersonServiceTest(
+  @MockBean val assessRisksAndNeedsGateway: AssessRisksAndNeedsGateway,
+  @MockBean val getPersonService: GetPersonService,
+  private val getRiskPredictorScoresForPersonService: GetRiskPredictorScoresForPersonService,
+) : DescribeSpec(
+  {
+    val pncId = "1234/56789B"
+    val nomisNumber = "Z99999ZZ"
+    val crn = "X123456"
```

Comment:
I think we should refer to CRN to `deliusCrn` everywhere as that's how we're exposing it and to also be consistent with `nomisNumber`.

See https://ministryofjustice.github.io/hmpps-integration-api-docs/documentation/api/index.html#schema-identifiers.

![image](https://github.com/ministryofjustice/hmpps-integration-api/assets/42817036/13febe2b-cd7d-44ad-89da-ae45dd345b8c)

Reply:
Cool, good call